recipe/meta.yaml

```diff
@@ -44,7 +45,7 @@ requirements:
     - maturin >=1.2.1,<2
```

Comment:
```suggestion
    - maturin >=1.3.2,<2
```

Reply:
This was already updated, but got reverted in https://github.com/conda-forge/polars-feedstock/pull/240/commits/020bce2dfdceecdce48c89a465ebb16ff64d6869, @borchero was this intended?

Reply:
Once the remaining CI jobs are completed - I don't expect any failures - I'd go ahead and merge this as is, if there are no objections. I can bump the `maturin` dependency in a new PR. This way we can avoid having to wait for another round of CI jobs until this version becomes available via conda-forge. 

Please let me know, if that works for everyone.

Reply:
When the CI is through, I'd be fine with bumping the pin and merging without waiting for another whole round of builds.

Reply:
Ok, then I will bump on this branch, but will not wait for the new round of CI to finish. I'll just merge this after the push.

Reply:
I don't know why that disappeared, I didn't (want to) touch it :eyes: